### PR TITLE
added_fitness_website

### PR DESCRIPTION
**Pull request by aditya-bhaumik**
Closes: #400

- **Title** :  Fitness Website
- **Name:** : Aditya Bhaumik
- **Idenitfy yourself:** :  Gssoc

### Describe the add-ons or changes you've made 📃

A fitness website provides users with resources and tools for improving their physical health, including workout plans, exercise tutorials,  and tracking features to help users achieve their fitness goals.

### How Has This Been Tested? ⚙️

I have runned this on my local machine.
Also i made sure that the website is responsive and contains good animations 

### Screenshots 📷

image

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Note to reviewers 📄

@Avdhesh-Varshney Please review my pull request for the completed fitness website  let me know if i have missed something i have ensured this time to make the no of commits less

### CODE REVIEW

File: database/vanilla.json
Comment (Avdhesh-Varshney) [inline database/vanilla.json:1]: don't disturb the alignment

File: Vanilla-JS-Projects/Intermediate/Fitness-Website/README.md
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/Intermediate/Fitness-Website/README.md:1]: properly follow the readme template

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@aditya-bhaumik resolve the merge conflicts

**Comment by aditya-bhaumik** [comment]
@Avdhesh-Varshney i have resolved the merge conflicts and made the changes requested

**Review by Avdhesh-Varshney — APPROVED** [review]
@aditya-bhaumik PR Approved 🎉

**Pull request merged**